### PR TITLE
feat: allow extra args to be passed to nextjs cli

### PR DESCRIPTION
**Pull request by julianklumpers**
Closes: -

### What are the changes and their implications?

Allow extra args to be passed to the nextjs CLI. For example in my case: `blitz dev -p 3000 --experimental-https`. The `--experimental-https` will be passed down to he nextjs cli. 

## Bug Checklist

- [x] Changeset added (run `pnpm changeset` in the root directory)
- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [x] Changeset added (run `pnpm changeset` in the root directory)
- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `main` branch](https://github.com/blitz-js/blitzjs.com))

### CODE REVIEW

File: packages/blitz/src/cli/commands/next/start.ts [hunk 1]
```diff
@@ -17,12 +17,15 @@ const start: CliCommand = async () => {
...
 
+  const extraArgs = nextArgs["_"].filter((arg) => arg !== "start")
+
   const config: ServerConfig = {
     rootFolder: process.cwd(),
     port: nextArgs["--port"],
     hostname: nextArgs["--hostname"],
     inspect: nextArgs["--inspect"],
     env: process.env.NODE_ENV === "production" ? "prod" : "dev",
+    extraArgs,
```
Comment (flybayer) [inline packages/blitz/src/cli/commands/next/start.ts:28]: hey thank you! Been wanting this for awhile.

Few things

- we need to make sure extraArgs is not also including ones we explicitly use (like `--inspect`)
- we should remove our use of `--hostname` and `--port` while we are here, they are legacy and no longer needed

Comment (flybayer) [reply packages/blitz/src/cli/commands/next/start.ts:28]: hi @julianklumpers, are you able to make these changes?

Comment (julianklumpers) [reply packages/blitz/src/cli/commands/next/start.ts:28]: yes, sorry for my late reply. Will try to implement this very soon.

**Comment by flybayer** [comment]
Hey, I'm going to close this for now as it's gone cold. Feel free to fix and re-open :)

**Pull request closed without merging**